control-plane/api-gateway/binding/validation.go

```diff
@@ -41,6 +41,45 @@ var (
 		gwv1beta1.Kind("HTTPRoute"): {},
 		gwv1beta1.Kind("TCPRoute"):  {},
 	}
+
+	allSupportedTLSVersions = map[string]struct{}{
+		"TLS_AUTO": {},
+		"TLSv1_0":  {},
+		"TLSv1_1":  {},
+		"TLSv1_2":  {},
+		"TLSv1_3":  {},
+	}
+
+	allTLSVersionsWithConfigurableCipherSuites = map[string]struct{}{
+		// Remove these two if Envoy ever sets TLS 1.3 as default minimum
```

Comment:
what `two` is this referring to? this set and the above set?

Reply:
I'll make this more explicit
```suggestion
		// Remove "" and "TLS_AUTO" if Envoy ever sets TLS 1.3 as default minimum
```